core/src/main/kotlin/io/fritz2/dom/html/elements.kt

```diff
@@ -7,24 +7,71 @@ import kotlinx.coroutines.ExperimentalCoroutinesApi
+ */
+class A : Tag<HTMLAnchorElement>("a"), WithText<HTMLAnchorElement> {
+    var target: Flow<String> by AttributeDelegate
+    var download: Flow<String> by AttributeDelegate
+    var ping: Flow<String> by AttributeDelegate
+    var rel: Flow<String> by AttributeDelegate
+    var hreflang: Flow<String> by AttributeDelegate
+    var type: Flow<String> by AttributeDelegate
+    var text: Flow<String> by AttributeDelegate
+    var referrerPolicy: Flow<String> by AttributeDelegate
+    var href: Flow<String> by AttributeDelegate
+    var protocol: Flow<String> by AttributeDelegate
+    var username: Flow<String> by AttributeDelegate
+    var password: Flow<String> by AttributeDelegate
+    var host: Flow<String> by AttributeDelegate
+    var hostname: Flow<String> by AttributeDelegate
+    var port: Flow<String> by AttributeDelegate
+    var pathname: Flow<String> by AttributeDelegate
+    var search: Flow<String> by AttributeDelegate
+    var hash: Flow<String> by AttributeDelegate
 }
 
-class Span : Tag<HTMLSpanElement>("span"), WithText<HTMLSpanElement>
-
-class Ul : Tag<HTMLUListElement>("ul")
+@ExperimentalCoroutinesApi
+@FlowPreview
+/**
+ * Exposes the JavaScript [HTMLAreaElement](https://developer.mozilla.org/en/docs/Web/API/HTMLAreaElement) to Kotlin
+ */
+class Area : Tag<HTMLAreaElement>("area"), WithText<HTMLAreaElement> {
+    var alt: Flow<String> by AttributeDelegate
+    var coords: Flow<String> by AttributeDelegate
+    var shape: Flow<String> by AttributeDelegate
+    var target: Flow<String> by AttributeDelegate
+    var download: Flow<String> by AttributeDelegate
+    var ping: Flow<String> by AttributeDelegate
+    var rel: Flow<String> by AttributeDelegate
+    var referrerPolicy: Flow<String> by AttributeDelegate
+    var href: Flow<String> by AttributeDelegate
+    var protocol: Flow<String> by AttributeDelegate
+    var username: Flow<String> by AttributeDelegate
+    var password: Flow<String> by AttributeDelegate
+    var host: Flow<String> by AttributeDelegate
+    var hostname: Flow<String> by AttributeDelegate
+    var port: Flow<String> by AttributeDelegate
+    var pathname: Flow<String> by AttributeDelegate
+    var search: Flow<String> by AttributeDelegate
+    var hash: Flow<String> by AttributeDelegate
+}
 
-class Li : Tag<HTMLElement>("li"), WithText<HTMLElement>
+@ExperimentalCoroutinesApi
+@FlowPreview
+/**
+ * Exposes the JavaScript [HTMLBRElement](https://developer.mozilla.org/en/docs/Web/API/HTMLBRElement) to Kotlin
+ */
+class Br : Tag<HTMLBRElement>("br")
 
-//FIXME: use correct type for domNode - HtmlButtonElement here
 @ExperimentalCoroutinesApi
 @FlowPreview
+/**
+ * Exposes the JavaScript [HTMLButtonElement](https://developer.mozilla.org/en/docs/Web/API/HTMLButtonElement) to Kotlin
+ */
 class Button : Tag<HTMLButtonElement>("button"), WithText<HTMLButtonElement> {
     var autofocusFlow: Flow<Boolean> by AttributeDelegate
```

Comment:
why is this calls autofocusFlow? same for next line?

Reply:
Oh, I didn't saw that. Thanks for reporting. Fix that in next commit.